### DISCUSSION TITLE
list of object, how to pass only id to preselect value

### DISCUSSION THREAD

**Discussion by Rubon72** [shentao/vue-multiselect#1747] [Q&A]
<multiselect
      v-model="3"
      :label="itemText"
      :trackBy="itemValue"
      :options="[
        {
          id:null,
          name:'Defaul Country'
        },
        {
          id:3,
          name:'USA'
        }
      ]"
    />

**Comment by mattelen** [comment]
I haven't run your code, but I think you'll want your `trackBy` to be `id`, and your `label` to be `name`, based on your options. See if that would work
